migration/1708096213854-addProjectFraudEntity.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class addProjectFraudEntity1708096213854 implements MigrationInterface {
```

Comment:
why the specific number in the class name?

Reply:
Those are autogenerated by the framework typeorm when creating new migration, we use this command:
```
typeorm migration:create --dir ./migration --name addProjectFraudEntity
```
Its a number based on the date of creation of the migration.